performance_regression_test.py

```diff
@@ -42,9 +47,9 @@ def display_single_result(self, result):
                                           result['Total partitions'],
                                           result['Total errors']))
 
-    def get_test_xml(self, result):
+    def get_test_xml(self, result, test_name='simple_regression_test'):
```

Comment:
It's good to put the test_name as a parameter. At the beginning, we try to implement 3 subtests (write, read, mixed) in one function-test_simple_regression(). But read & mixed tests are more complex. So we add test_read() and test_mixed().

Let's clean test_simple_regression to only test write, I will send a PR today.